src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC, useState } from 'react';
+import { NavbarBrand } from './NavbarBrand';
+import { NavbarCollapse } from './NavbarCollapse';
+import { NavbarContext } from './NavbarContext';
+import { NavbarToggle } from './NavbarToggle';
+
+export type NavbarCompnentProps = {
+  open?: boolean;
```

Comment:
Can we make the name of this property clear? from the user perspective is this `menuOpen` or `sidebarOpen`...?